### ISSUE TITLE
Retrieving ES relevancy score?

### ISSUE THREAD

**Issue by coreyculler** [barseghyanartur/graphene-elastic#36]
Hello, first off let me say that I love this integration and that it has made integrating ES with Graphene very easy. However, I am curious if it is possible to retrieve the calculated relevancy score of each search result? If it is possible, how would this be done using this library?

**Comment by barseghyanartur** [comment]
It should be possible to expose `_score`. Let me know if you have trouble implementing this. Eventually, PRs are welcome.

**Comment by coreyculler** [comment]
How would this be done? Would I have to create a custom resolver to retrieve the `_score` attribute? Is that attribute going to be available on the parent object within an ElasticsearchObjectType resolver?

**Comment by barseghyanartur** [comment]
Yep. Most likely, through a filter backend. It should be done in a declarative, non-breaking manner. Somewhat, like in [highlight](https://github.com/barseghyanartur/graphene-elastic/blob/master/src/graphene_elastic/filter_backends/highlight/common.py), but much simpler.

**Comment by barseghyanartur** [comment]
Check this (not yet released, just pushed into master):

```python
from graphene import Node
from graphene_elastic import ElasticsearchObjectType
from graphene_elastic.filter_backends import (
    # ...
    ScoreFilterBackend,
    # ...
)
from search_index.documents import Post as PostDocument

class Post(ElasticsearchObjectType):

    class Meta:

        document = PostDocument
        interfaces = (Node,)
        filter_backends = [
            # ...
            ScoreFilterBackend,
            # ...
        ]

        # For `DefaultOrderingFilterBackend` backend
        ordering_defaults = (
            '_score',
            # ...
        )
```

Do make sure you have ordered results by `_score` (as shown in `ordering_defaults` above).

Query like this:

```graphql
query PostsQuery {
      allPostDocuments(
        	search:{query:"Budget"},
        	filter:{category:{value:"Elastic"}},
      ) {
        edges {
          node {
            id
            title
            category
            content
            createdAt
            score
          }
        }
      }
    }
```

Will produce results similar to these:

```json
{
  "data": {
    "allPostDocuments": {
      "edges": [
        {
          "node": {
            "id": "UG9zdDpMNnBiV1hNQjhYRzdJclZ2X20waA==",
            "title": "Budget.",
            "category": "Elastic",
            "content": "Bed television public teacher behind human up.\nMind anyone politics ball cost wife try adult. College work for.\nPlay five ten not sort energy.\nCommon word behind spring. All behind voice policy.",
            "createdAt": "1973-03-12T00:00:00",
            "score": 20.420774
          }
        },
       ]
    }
  }
}
```

**Comment by coreyculler** [comment]
Oh wow, thanks for the quick code update! I'll try it out tomorrow.

**Comment by barseghyanartur** [comment]
@coreyculler:

Some more work is done on this. It's now possible (in master) to allow sorting on score (if added as option to the `ordering_fields`). See updated example.

**Comment by barseghyanartur** [comment]
Released in 0.6.3.
